Source/EventFlow.MsSql/ReadStores/MssqlReadModelStore.cs

```diff
@@ -142,6 +142,7 @@ static MssqlReadModelStore()
                     readModelEnvelope,
                     cancellationToken)
                 .ConfigureAwait(false);
+            if (readModelEnvelope == null) return;
```

Comment:
@frankebersoll I would like to finish this PR soon, but I'm very unsure how well I managed to do the implementation, especially this part (return null signals no updates to read models). If you have the time I would very much like a second pair of eyes on this.

Reply:
Same here. I think I'll go with your suggestion regarding the `ReadModelUpdateResult` as this part will closely integrate with read model stores and being explicit here makes it easier to create integrations instead of having some secret meaning to specific values.

Thanks for the input,

Reply:
Trying a `IsModified` on the envelope

Reply:
I'm not sure... The envelope is also used "externally" by IReadModelStore users, mainly query handlers. I won't expect an "IsModified" on the envelope there either.

Reply:
@frankebersoll Didn't see your response, sorry. Will have a look